Src/CSharpier.VSCode/src/CSharpierProcessServer.ts

```diff
@@ -19,8 +20,9 @@ export class CSharpierProcessServer implements ICSharpierProcess2 {
 
         this.logger.debug("Warm CSharpier with initial format");
         // warm by formatting a file twice, the 3rd time is when it gets really fast
-        this.formatFile("public class ClassName { }", "/Temp/Test.cs").then(() => {
-            this.formatFile("public class ClassName { }", "/Temp/Test.cs");
+        // make sure we give a path that should not exist to avoid any errors when trying to find config files etc.
+        this.formatFile("public class ClassName { }", `/${Date.now()}/Test.cs`).then(() => {
```

Comment:
if the goal is for the directory to not exists, why not just use `""`?

Reply:
If I remember correctly, csharpier is unhappy if you do that. There should probably just be an option for "no file path" and 0.28.0 would have been the time to add it. Dammit.